components/kyma-environment-broker/internal/broker/instance_update_test.go

```diff
@@ -184,45 +184,20 @@ func TestUpdateEndpoint_UpdateInstanceWithWrongActiveValue(t *testing.T) {
 }
 
 func fixProvisioningOperation(id string) internal.ProvisioningOperation {
-	return internal.ProvisioningOperation{
-		Operation: internal.Operation{
-			ID:         id,
-			CreatedAt:  time.Now(),
-			InstanceID: instanceID,
-			ProvisioningParameters: internal.ProvisioningParameters{
-				ErsContext: internal.ERSContext{
-					ServiceManager: &internal.ServiceManagerEntryDTO{
-						Credentials: internal.ServiceManagerCredentials{
-							BasicAuth: internal.ServiceManagerBasicAuth{
-								Username: "u",
-								Password: "p",
-							},
-						},
-					},
-				},
-			},
-		},
-	}
+	provisioningOperation := fixture.FixProvisioningOperation(id, instanceID)
+	provisioningOperation.ProvisioningParameters.ErsContext.ServiceManager.Credentials.BasicAuth.Username = "u"
+	provisioningOperation.ProvisioningParameters.ErsContext.ServiceManager.Credentials.BasicAuth.Password = "p"
+	provisioningOperation.ProvisioningParameters.ErsContext.ServiceManager.URL = ""
+
+	return provisioningOperation
 }
 
 func fixSuspensionOperation() internal.DeprovisioningOperation {
-	return internal.DeprovisioningOperation{
-		Operation: internal.Operation{
-			CreatedAt:  time.Now(),
-			InstanceID: instanceID,
-			ProvisioningParameters: internal.ProvisioningParameters{
-				ErsContext: internal.ERSContext{
-					ServiceManager: &internal.ServiceManagerEntryDTO{
-						Credentials: internal.ServiceManagerCredentials{
-							BasicAuth: internal.ServiceManagerBasicAuth{
-								Username: "u",
-								Password: "p",
-							},
-						},
-					},
-				},
-			},
-		},
-		Temporary: true,
-	}
+	deprovisioningOperation := fixture.FixDeprovisioningOperation("id", instanceID)
+	deprovisioningOperation.ProvisioningParameters.ErsContext.ServiceManager.Credentials.BasicAuth.Username = "u"
```

Comment:
this could be set directly in `fixture.go` like this:

```golang
func FixServiceManagerEntryDTO() *internal.ServiceManagerEntryDTO {
	return &internal.ServiceManagerEntryDTO{
		Credentials: internal.ServiceManagerCredentials{
			BasicAuth: internal.ServiceManagerBasicAuth{
				Username: "u",
				Password: "p",
			},
		},
		URL: "https://service-manager.local",
	}
}
```
and even better if username, password and URL values are moved to `const` section in `fixture.go` too. Then we could reuse those when working with the values in tests, because these are only used directly when comparing fields in `assertServiceManagerCreds()` function which asserts that `Handle()` method was called - in fake handler this method sets two fields in a struct with data coming from `Update` call in KEB:
```golang
func (h *handler) Handle(inst *internal.Instance, ers internal.ERSContext) error {
	h.Instance = *inst
	h.ersContext = ers
	return nil
}
```

so, the whole `ERSContext` can be set globally in `Operation` when creating it using `fixture` package - after changing this, please adjust `assertServiceManagerCreds` function to use the same values when comparing fields.